### PR TITLE
Adds total machine cost info

### PR DESCRIPTION
**Pull request by KhizerJaan**

### CODE REVIEW

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-84-1c9ea2cf`

**Review by usamaahmadkhan — APPROVED** [review]

File: docs/content/sre/machinepools/machinepools.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ The control plane, which is composed of control plane machines (also known as th
 |   | vCPU | Memory |  Cost/mo<sup id="fnref1"><a href="#fn1" rel="footnote">*</a></sup> |
 |---|---|---|---|
 | Amazon EC2  | 8  | 64 GiB  | 400 USD  |
-|  Binero | 8 | 48 GB | 2079.88 SEK  |
+|  Binero | 8 | 48 GB | 2080 SEK  |
```
Comment (usamaahmadkhan) [inline docs/content/sre/machinepools/machinepools.md]: we use 8x32 GB not 8x64. Plz change this for both

**Review by usamaahmadkhan — CHANGES_REQUESTED** [review]

**Comment by stakater-user** [comment]
@KhizerJaan Image is available for testing. `docker pull nexus-docker-stakater-nexus.apps.preprod.jlvwjls8.kubeapp.cloud/stakater/stakaterclouddocs:SNAPSHOT-PR-84-bb9b49b8`

**Comment by karl-johan-grahn** [comment]
redundant

**Pull request closed without merging**